metapool/count.py

```diff
@@ -225,16 +227,99 @@ def minimap2_counts(run_dir, metadata):
                       'non_host_reads', _parse_samtools_counts)
 
 
+def count_sequences_in_fastq(file_path):
+    variants = ['illumina1.8', 'illumina1.3', 'sanger']
+
+    for variant in variants:
+        try:
+            # see https://scikit.bio/docs/latest/generated/skbio.io.format.
+            # fastq.html#quality-score-variants for more information on
+            # variants. Variant is needed to correctly interpret a sequence's
+            # quality range. We don't explicitly need quality-ranges for this
+            # application but we unfortunately we cannot omit it.
+            sequences = skb_read(file_path, format='fastq',
+                                 variant=variant)
+            return sum(1 for sequence in sequences)
+        except ValueError:
+            # assume ValueError is because the correct variant wasn't
+            # specified.
+            pass
```

Comment:
FWIW, other plugins use [fqtools count](https://github.com/qiita-spots/qtp-sequencing/blob/master/qtp_sequencing/validate.py#L207) as [is normally faster than other solutions](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4908325/). 

Anyway, just wanted to leave here in case you want it to change or add an issue to fix in the future. This means that I'm OK with the current solution.

Reply:
ty! I will review and potentially add an issue to fix in the future.